typescript/hardhat/test/testAbacusDeploy.test.ts

```diff
@@ -38,35 +38,57 @@ describe("TestAbacusDeploy", async () => {
     ).to.emit(remoteOutbox, "Dispatch");
   });
 
-  it("processes outbound messages for a single domain", async () => {
-    const responses = await abacus.processOutboundMessages(localDomain);
-    expect(responses.get(remoteDomain)!.length).to.equal(1);
-    const [_, index] = await abacus.outbox(localDomain).latestCheckpoint();
-    expect(index).to.equal(1);
+  describe("without having called checkpoint", () => {
```

Comment:
It looks like prettier is either not running on this file or is not properly configured to use single quotes